src/rum.js

```diff
@@ -52,7 +25,22 @@ if (!rumConfig.serverUrl) {
 if (!rumConfig.serviceVersion) {
     rumConfig.serviceVersion = serviceVersion
 }
-rumConfig.logLevel = 'debug'
+
+rumConfig.active = process.env.REACT_APP_ELASTIC_APM_ACTIVE === 'true' || true
+rumConfig.breakdownMetrics = process.env.REACT_APP_ELASTIC_APM_BREAKDOWN_METRICS === 'true' || false
+rumConfig.centralConfig = process.env.REACT_APP_ELASTIC_APM_CENTRAL_CONFIG === 'true' || false
+rumConfig.disableInstrumentations = stringToList(process.env.REACT_APP_ELASTIC_APM_DISABLE_INSTRUMENT) || []
+rumConfig.distributedTracing = process.env.REACT_APP_ELASTIC_APM_DISTRIBUTED_TRACING === 'true' || true
+rumConfig.distributedTracingOrigins = stringToList(process.env.ELASTIC_APM_DISTRIBUTED_TRACING_ORIGINS)
+rumConfig.environment = process.env.REACT_APP_ELASTIC_APM_ENVIRONMENT || 'production'
+rumConfig.errorThrottleInterval = Number(process.env.REACT_APP_ELASTIC_APM_ERROR_THROTTLE_INTERVAL) || 30000
+rumConfig.errorThrottleLimit = Number(process.env.REACT_APP_ELASTIC_APM_ERROR_THROTTLE_LIMIT) || 20
```

Comment:
Are these actual RUM Agent options? They aren't in https://www.elastic.co/guide/en/apm/agent/rum-js/current/configuration.html

Reply:
As Trent mentioned, we have only eventsThrottle which includes transaction and error events - https://www.elastic.co/guide/en/apm/agent/rum-js/current/configuration.html#events-limit

Reply:
I have checked an old documentation https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/configuration.html 🤦